hazelcast/src/main/java/com/hazelcast/monitor/impl/NearCacheStatsImpl.java

```diff
@@ -127,13 +133,41 @@ public double getRatio() {
     }
 
     @Override
+    public long getEvictions() {
+        return evictions;
+    }
+
+    public void setEvictions(long evictions) {
+        EVICTIONS.set(this, evictions);
+    }
+
+    public void incrementEvictions() {
+        EVICTIONS.incrementAndGet(this);
+    }
+
+    @Override
+    public long getExpirations() {
+        return expirations;
+    }
+
+    public void setExpirations(long expirations) {
```

Comment:
Same  here. 

If this isn't an issue, then probably best to document that it isn't an issue so that future readers don't get their internal alarms triggered.